src/averageAges.js

```diff
@@ -14,12 +14,28 @@
  *
  * @return {number}
  */
+function avereageAge(peopleArray, ageArray) {
+  const sumAge = ageArray.reduce((x, sum) => x + sum);
+  const resAverageAge = sumAge / peopleArray.length;
+
+  return resAverageAge;
+}
+
 function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
```

Comment:
remove comments

Reply:
I didn't understand what was wrong.  I don't use if and for each.  I am using map, filter, find.